### PR TITLE
feat(eth-sender): add time_in_mempool_cap config

### PR DESCRIPTION
**Pull request by tomg10**
Configuration parameter for time_in_mempool_cap, cap for time_in_mempool in eth-sender fee model + default values for parameter_a and parameter_b for GasAdjuster

### CODE REVIEW

File: core/node/eth_sender/src/eth_fees_oracle.rs [hunk 1]
```diff
@@ -83,7 +84,7 @@ impl GasAdjusterFeesOracle {
         time_in_mempool: u32,
     ) -> Result<EthFees, EthSenderError> {
         // cap it at 6h to not allow nearly infinite values when a tx is stuck for a long time
```
Comment (perekopskiy) [inline core/node/eth_sender/src/eth_fees_oracle.rs:86]: update comment

File: core/lib/config/src/configs/eth_sender.rs [hunk 2]
```diff
@@ -127,6 +128,10 @@ pub struct SenderConfig {
     /// special mode specifically for gateway migration to decrease number of non-executed batches
     #[serde(default = "SenderConfig::default_tx_aggregation_only_prove_and_execute")]
     pub tx_aggregation_only_prove_and_execute: bool,
+
+    /// Cap of time in mempool for price calculations
+    #[serde(default = "SenderConfig::default_time_in_mempool_cap")]
+    pub time_in_mempool_cap: u32,
```
Comment (perekopskiy) [inline core/lib/config/src/configs/eth_sender.rs:134]: nit: include `_sec` in name

Comment (tomg10) [reply core/lib/config/src/configs/eth_sender.rs:134]: it's not in seconds, but in l1 blocks, I've refactored all time_in_mempool references to time_in_mempool_in_l1_blocks to increase readability

File: core/node/eth_sender/src/eth_fees_oracle.rs (hunk 1 above)
Comment (tomg10) [reply core/node/eth_sender/src/eth_fees_oracle.rs]: corrected

**Review by perekopskiy — APPROVED** [review]

**Pull request merged**